eogrow/utils/testing.py

```diff
@@ -180,6 +180,11 @@ def _calculate_numpy_stats(self, raster: np.ndarray) -> JsonDict:
                 for name, operation in self._STATS_OPERATIONS.items()
             }
 
+            subsample = self._get_subsample(finite_values)
+            stats["subsample_basic_stats"] = {
+                name: self._prepare_value(operation(subsample)) for name, operation in self._STATS_OPERATIONS.items()
+            }
```

Comment:
Maybe it would be a bit nicer if you moved all more code into the the submethod that currently has only `2` lines. You could have something like this:

```Python
stats["subsample_basic_stats"] = self._get_subsample_stats(finite_values)
```

Reply:
done